packages/eslint-plugin/src/rules/no-other-module-relative-import.test.ts

```diff
@@ -0,0 +1,40 @@
+import { RuleTester } from "eslint";
+
+import noOtherModuleRelativeImport from "./no-other-module-relative-import";
+
+const ruleTester = new RuleTester({
+    parser: require.resolve("@typescript-eslint/parser"),
+});
+
+const errors = [{ message: "Avoid relative import from other module" }];
+
+const options = [{ sourceRoot: "./src", sourceRootAlias: "@src" }];
+
+ruleTester.run("no-other-module-relative-import", noOtherModuleRelativeImport, {
+    valid: [
+        {
+            code: `import Bar from "@src/moduleb/Bar";`,
+            filename: `${process.cwd()}/src/modulea/Foo.ts`,
+            options,
+        },
+        { code: `import Bar from "../Bar";`, filename: `${process.cwd()}/src/modulea/sub/Foo.ts`, options },
+        { code: `import Bar from "xx/bar";`, filename: `${process.cwd()}/src/modulea/Foo.ts`, options },
```

Comment:
Why is this valid? Is this for unknown source root aliases?

Reply:
npm package :)